src/Compilers/CSharp/Portable/Declarations/DeclarationTreeBuilder.cs

```diff
@@ -367,6 +368,9 @@ private RootSingleNamespaceDeclaration CreateRootSingleNamespaceDeclaration(Comp
 
             var globalAliasedQuickAttributes = GetQuickAttributes(compilationUnit.Usings, global: true);
 
+            CheckUsings(diagnostics, compilationUnit.Usings);
```

Comment:
>CheckUsings [](http://example.com/codeflow?start=12&length=11)

nit: consider slightly more specific names like `CheckFeatureAvailabilityForUsings` and `CheckFeatureAvailabilityForExterns`

Reply:
Done.